### PR TITLE
FIX: Matching new polars version

### PR DESCRIPTION
**Pull request by akmalsoliev**

### CODE REVIEW

File: src/timezone.rs [hunk 1]
```diff
@@ -35,6 +35,11 @@ fn naive_local_to_naive_utc_in_new_time_zone(
             Ambiguous::Raise => {
                 polars_bail!(ComputeError: "datetime '{}' is ambiguous in time zone '{}'. Please use `ambiguous` to tell how it should be localized.", ndt, to_tz)
             }
+            Ambiguous::Null => {
+                polars_bail!(ComputeError: 
+                "datetime '{}' is ambiguous in time zone '{}'. Please use `ambiguous` to tell how it should be localized.",
```
Comment (MarcoGorelli) [inline src/timezone.rs:40]: copy-pasta 😉  this is the non-existent branch, not the ambiguous one

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: src/timezone.rs (hunk 1 above)
Comment (akmalsoliev) [reply src/timezone.rs:40]: > copy-pasta 😉 this is the non-existent branch, not the ambiguous one
😪 brain is shutting down

Comment (akmalsoliev) [reply src/timezone.rs:40]: > copy-pasta 😉 this is the non-existent branch, not the ambiguous one

how about this:
`datetime '{}' does not exist or is invalid in time zone '{}'.`

Comment (MarcoGorelli) [reply src/timezone.rs:40]: Oh wait sorry it was my comment that was wrong

This should say that Ambiguous::null is not yet supported here (we can support it in a separate pr)

Comment (akmalsoliev) [reply src/timezone.rs]: > Oh wait sorry it was my comment that was wrong
> 
> This should say that Ambiguous::null is not yet supported here (we can support it in a separate pr)

Put under `unimplemented!` for now

**Review by MarcoGorelli — APPROVED** [review]
Thanks @akmalsoliev!

**Pull request merged**